.eslintrc.yml

```diff
@@ -664,3 +664,27 @@ overrides:
       import/no-nodejs-modules: off
       import/no-commonjs: off
       no-console: off
+  - files: 'docusaurus.config.js'
+    rules:
+      import/no-commonjs: off
+      import/no-nodejs-modules: off
+      import/no-extraneous-dependencies: off
+      internal-rules/only-ascii: off
+      node/no-unpublished-require: off
+  - files: 'docs/**'
+    rules:
+      import/no-commonjs: off
+      import/unambiguous: off
+      import/no-extraneous-dependencies: off
+      node/no-unpublished-import: off
+      node/no-missing-import: off
+      # Ignore docusarus related webpack aliases
+      import/no-unresolved:
+        ['error', { 'ignore': ['^@theme', '^@docusaurus', '^@generated'] }]
+    extends:
+      - 'plugin:react/recommended'
+    parserOptions:
+      sourceType: module
+    parser: 'babel-eslint'
+    plugins:
+      - 'react'
```

Comment:
```suggestion
    parser: 'babel-eslint'
    parserOptions:
      sourceType: module
    plugins:
      - 'react'
    extends:
      - 'plugin:react/recommended'
    rules:
      import/no-commonjs: off
      import/unambiguous: off
      import/no-extraneous-dependencies: off
      node/no-unpublished-import: off
      node/no-missing-import: off
      # Ignore docusarus related webpack aliases
      import/no-unresolved:
        ['error', { 'ignore': ['^@theme', '^@docusaurus', '^@generated'] }]
```

Reply:
order in overrides should much order and in the main config and also should make a general sense (e.g. `parserOptions` should be after `parser` not before)